src/app-nordigen/services/nordigen-service.js

```diff
@@ -15,11 +15,14 @@ import * as nordigenNode from 'nordigen-node';
 import * as uuid from 'uuid';
 import config from '../../load-config.js';
 import jwt from 'jws';
+import { secretsService } from '../../services/secrets-service.js';
 
 const NordigenClient = nordigenNode.default;
 const nordigenClient = new NordigenClient({
-  secretId: config.nordigen?.secretId,
-  secretKey: config.nordigen?.secretKey,
+  secretId:
+    secretsService.get('nordigen_secretId') || config.nordigen?.secretId,
```

Comment:
It feels like it would be super easy to typo these secret names. What do you think about defining a `secretNames` object with all valid secret names so that you could do `secretsService.get(secretNames.nordigen.secretId)`?